hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -359,7 +359,7 @@ bool MDMParser::_atOk(void)
 
 int MDMParser::waitFinalResp(_CALLBACKPTR cb /* = NULL*/,
                              void* param /* = NULL*/,
-                             system_tick_t timeout_ms /*= 5000*/)
+                             system_tick_t timeout_ms /*= 10000*/)
```

Comment:
Any reasoning behind the change?

Reply:
Fixed the comment because we are using `10000 msec` 
please see this? https://github.com/particle-iot/device-os/blob/30a8f2e409d3351bfcdff85b4f0efb53fe8eddae/hal/src/electron/modem/mdm_hal.h#L454